internal/service/crdparser/crdparser.go

```diff
@@ -41,12 +42,12 @@ type Resource struct {
 	} `yaml:"spec"`
 }
 
-func (s *Service) IsCRDClusterScoped(crPath, manifestPath string) (bool, error) {
-	if crPath == "" {
+func (s *Service) IsCRDClusterScoped(paths *types.ResourcePaths) (bool, error) {
+	if paths.DefaultCR == "" {
 		return false, nil
 	}
 
-	crData, err := s.fileSystem.ReadFile(crPath)
+	crData, err := s.fileSystem.ReadFile(paths.DefaultCR)
 	if err != nil {
 		return false, fmt.Errorf("error reading CRD file: %w", err)
```

Comment:
this existing error message is mislead, it's not read CRD file, but default CR file